packages/desktop-client/src/components/mobile/transactions/TransactionList.jsx

```diff
@@ -119,6 +145,244 @@ export function TransactionList({
+    style: {
+      ...styles.mobileMenuItem,
+      color: 'currentColor',
+      height: styles.mobileMinHeight,
+    },
+    activeStyle: {
+      color: 'currentColor',
+    },
+    hoveredStyle: {
+      color: 'currentColor',
+    },
+  };
+
+  const allTransactionsAreLinked = useMemo(() => {
+    return transactions
+      .filter(t => selectedTransactions.includes(t.id))
+      .every(t => t.schedule);
+  }, [transactions, selectedTransactions]);
+
+  const isMoreThanOne = selectedTransactions.length > 1;
+
+  return (
+    <View
+      style={{
+        backgroundColor: theme.floatingActionBarBackground,
+        color: theme.floatingActionBarText,
+        position: 'fixed',
+        bottom: 10,
+        margin: '0 10px',
+        width: '95vw',
+        height: 60,
+        zIndex: 100,
+        borderRadius: 8,
+        border: `1px solid ${theme.floatingActionBarBorder}`,
+        ...style,
+      }}
+    >
+      <View
+        style={{
+          flex: 1,
+          padding: 8,
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+        }}
+      >
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+          }}
+        >
+          <Button
+            type="bare"
+            {...buttonProps}
+            style={{ ...buttonProps.style, marginRight: 4 }}
+            onClick={() => onClearSelectedTransactions?.()}
+          >
+            <SvgDelete width={10} height={10} />
+          </Button>
+          <Text style={styles.mediumText}>
+            {selectedTransactions.length}{' '}
+            {isMoreThanOne ? 'transactions' : 'transaction'} selected
+          </Text>
+        </View>
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'flex-end',
+            gap: 4,
+          }}
+        >
+          <Button
+            type="bare"
+            ref={editMenuTriggerRef}
+            aria-label="Edit fields"
+            onClick={() => {
+              setIsEditMenuOpen(true);
+            }}
+            {...buttonProps}
+          >
+            Edit
+          </Button>
+
+          <Popover
+            triggerRef={editMenuTriggerRef}
+            isOpen={isEditMenuOpen}
+            onOpenChange={() => setIsEditMenuOpen(false)}
+            style={{ width: 200 }}
+          >
+            <Menu
+              getItemStyle={getMenuItemStyle}
+              style={{ backgroundColor: theme.floatingActionBarBackground }}
+              onMenuSelect={type => {
+                onEdit?.(type, selectedTransactions);
+                setIsEditMenuOpen(false);
+              }}
+              items={[
+                // Add support later on.
+                // Pikaday doesn't play well will mobile.
+                // We should consider switching to react-aria date picker.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏We definitely should do that! 😅